tests/test_math.py

```diff
@@ -752,6 +752,35 @@ def test_matrix_44():
         assert "Matrix44 should only be a 2D array of shape (4,4)." in str(val.value)
 
 
+def test_create_matrix_from_rotation():
+    direction_x = Vector3D([1, 0, 0])
+    direction_y = Vector3D([0, 1, 0])
+
+    expected_matrix = Matrix44([[1, 0, 0, 0], [0, 1, 0, 0], [0, 0, 1, 0], [0, 0, 0, 1]])
+    rotation_matrix = Matrix44.create_matrix_from_rotation(direction_x, direction_y)
+    assert np.array_equal(expected_matrix, rotation_matrix)
+
+
+def test_create_matrix_from_translation():
+    origin = Vector3D([1, 2, 3])
+
+    expected_matrix = Matrix44([[1, 0, 0, 1], [0, 1, 0, 2], [0, 0, 1, 3], [0, 0, 0, 1]])
+    translation_matrix = Matrix44.create_matrix_from_translation(origin)
```

Comment:
Origin looks odd.. shouldn't it be a Point3D?

Reply:
might as well, thanks you've got a point there:)